packages/st2/rpm/st2.spec

```diff
@@ -65,15 +65,15 @@ Conflicts: st2common
   %defattr(-,root,root,-)
   %{_bindir}/*
   %config(noreplace) %{_sysconfdir}/logrotate.d/st2
-  %config(noreplace) %{_sysconfdir}/st2/*
-  %{_datadir}/python/%{venv_name}
-  %{_datadir}/doc/st2/examples
-  %attr(755, %{svc_user}, %{svc_user}) %{_localstatedir}/log/st2
+  %config(noreplace) %attr(640, %{svc_user}, %{svc_user}) %{_sysconfdir}/st2/htpasswd
+  /opt/stackstorm/%{venv_name}
   /opt/stackstorm/packs/core
   /opt/stackstorm/packs/linux
   /opt/stackstorm/packs/packs
-  %attr(640, %{svc_user}, %{svc_user}) %{_sysconfdir}/st2/htpasswd
+  /opt/stackstorm/packs/examples
```

Comment:
Shouldn't be directory for packs examples `/opt/stackstorm/st2/packs/examples`?
Since packs is part of `st2`.